### PR TITLE
feat(gatsby-starter-blog): Remove typography.js

### PR DESCRIPTION
**Pull request by LekoArts**
## Description

This removes `typography.js` from the "blog" starter and replaces it with normal CSS files.

Typography.js is proprietary, difficult to understand (if you never used it) and cumbersome to remove from the project. The starter shouldn't be too opinionated and include it by default. With normal CSS files users can quickly remove all styling and replace it with their own. This problem was often mentioned by users.

The overall style didn't change much (a complete redesign can be done in the future) so the custom CSS replicates the style.

### Screenshots

image

image

image

image

### CODE REVIEW

File: starters/blog/content/blog/hello-world/index.md [hunk 1]
```diff
@@ -20,3 +20,212 @@ Oh, and here's a great quote from this Wikipedia on
...
 
 ![Chinese Salty Egg](./salty_egg.jpg)
+
+You can also write code blocks here!
+
+```js
+const saltyDuckEgg = "chinese preserved food product"
+```
+
+| Number | Title                                    | Year |
```
Comment (LekoArts) [inline starters/blog/content/blog/hello-world/index.md:30]: Some boilerplate to test out all markdown combinations

File: starters/blog/src/components/layout.js [hunk 2]
```diff
@@ -1,62 +1,28 @@
...
-          style={{
-            boxShadow: `none`,
-            color: `inherit`,
-          }}
-          to={`/`}
-        >
-          {title}
-        </Link>
-      </h3>
+      <Link className="header-link-home" to="/">
```
Comment (LekoArts) [inline starters/blog/src/components/layout.js:17]: `h3` wasn't the correct tag for this link (it always came before the h1 of the page)

**Review by ascorbic — APPROVED** [review]
This is awesome

File: starters/blog/src/pages/index.js [hunk 3]
```diff
@@ -24,28 +27,21 @@ const BlogIndex = ({ data, location }) => {
...
-                style={{
-                  marginBottom: rhythm(1 / 4),
-                }}
-              >
-                <Link
-                  style={{ boxShadow: `none` }}
-                  to={post.fields.slug}
-                  itemProp="url"
-                >
+              <h2>
```
Comment (LekoArts) [inline starters/blog/src/pages/index.js:40]: Correct order of heading tags was `h2` here

**Pull request merged**